node_modules/oae-principals/lib/test/util.js

```diff
@@ -468,3 +469,96 @@ var getAllGroupMembers = module.exports.getAllGroupMembers = function(restCtx, g
         return getAllGroupMembers(restCtx, groupId, opts, callback, _nextToken, _members, _responses);
     });
 };
+
+/*
+ * Sign into a guest account for a given guest token
+ *
+ * @param  {String}         token                       The token that can be used to sign into a guest account
+ * @param  {Function}       callback                    Standard callback function
```

Comment:
These aren't standard callback functions as they don't contain an error as the first parameter